frontend/src/routes/(authenticated)/org/list/+page.svelte

```diff
@@ -0,0 +1,105 @@
+<script lang="ts">
+  import type { OrgListPageQuery } from '$lib/gql/types';
+  import t, { date, number } from '$lib/i18n';
+  import { Page } from '$lib/layout';
+  import type { PageData } from './$types';
+
+  export let data: PageData;
+  $: orgs = data.orgs;
+
+  type OrgList = OrgListPageQuery['orgs']
+
+  type Column = 'name' | 'users' | 'created_at';
+  let sortColumn = 'created_at' as Column;
+  type Dir = 'ascending' | 'descending';
+  let sortDir = 'ascending' as Dir;
+
+  function swapSortDir(): void {
+    sortDir = sortDir == 'ascending' ? 'descending' : 'ascending';
+  }
+
+  function handleSortClick(clickedColumn: Column): void {
+    if (sortColumn === clickedColumn) {
+      swapSortDir();
+    } else {
+      sortColumn = clickedColumn;
+      sortDir = 'ascending';
+    }
+  }
+
+  function sortOrgs(orgs: OrgList, sortColumn: Column, sortDir: Dir): OrgList {
+    const data = [... orgs];
+    let mult = sortDir === 'ascending' ? 1 : -1;
+    data.sort((a, b) => {
+      if (sortColumn === 'users') {
+        return (a.members.length - b.members.length) * mult;
+      } else if (sortColumn === 'name') {
+        const comp = a.name < b.name ? -1 : a.name > b.name ? 1 : 0;
+        return comp * mult;
+      } else if (sortColumn === 'created_at') {
+        const comp = a.createdDate < b.createdDate ? -1 : a.createdDate > b.createdDate ? 1 : 0;
+        return comp * mult;
+      }
+      return 0;
+    });
+    return data;
+  }
+
+  $: displayOrgs = $orgs ? sortOrgs($orgs, sortColumn, sortDir) : [];
+</script>
+
+<!--
+TODO:
+
+* Sort options: name, created date, # users
+* Paging
+-->
+
+<Page wide>
+  <h1 class="text-3xl text-left grow max-w-full mb-4">Organisations</h1>
+  <div class="overflow-x-auto @container scroll-shadow">
+    <table class="table table-lg">
+      <thead>
+        <tr class="bg-base-200">
+          <th on:click={() => handleSortClick('name')} class="cursor-pointer">
```

Comment:
How about this to make it even more clear that they're clickable?
```suggestion
          <th on:click={() => handleSortClick('name')} class="cursor-pointer hover:bg-base-300">
```

Reply:
Done in commit 63e33167, along with the fix to make the sort icons no longer resize the table.